docs/TUTORIAL.md

```diff
@@ -1,3 +1,305 @@
 # Tutorial
 
-Coming soon!
+The **ksonnet** [readme][readme] 
+shows you how to create a default 
+`deployment.json` file that lets you deploy an nginx container 
+to an existing Kubernetes cluster. This tutorial shows you how to:
+
+* Modify the deployment using the built-in **ksonnet** mixins
+* Write a custom mixin library for Fluentd 
+* Add the custom Fluentd mixin to your application deployemtn
+
+## Prerequisites
```

Comment:
I like numbering the steps (so they seem sequential, rather than potentially unrelated sections) but I think I remember you having the opposite opinion ;P